index.html

```diff
@@ -7,5 +7,200 @@
 </head>
 <body>
 
+<!-------------------------------------------------- Content header --------------------------------------------------->
+<header class="outer-container column">
+    <nav class="navbar row">
+        <ul class="row">
```

Comment:
Het menu staat nu centered, lukte het niet om deze aan de rechterkant te plaatsen? Kleine tip: zet vanuit < nav> 
flexbox in de stijlregels met een justify content op flex-end. Daarna kan je in je styles document ul aanspreken ( nav ul {}  ) dan kan je gebruik maken om het menu te stylen. Dat geeft net wat meer bewegingsvrijheid, en betekent ook dat je niet de UL een class hoeft te geven.